src/main/java/org/folio/edge/controller/exception/ExceptionHandlerController.java

```diff
@@ -77,4 +83,25 @@ public Error handleException(Exception e) {
     return new ResponseEntity<>(new Error().error(INVALID_REQUEST_ERROR_TYPE), HttpStatus.valueOf(e.getStatus()));
   }
 
+  @ExceptionHandler(FeignException.class)
+  public ResponseEntity<InnReachResponseDTO> handleFeignStatusException(FeignException feignException) {
+    var status = HttpStatus.resolve(feignException.status());
```

Comment:
add logging of exception similar to what we do for other handlers

Reply:
Fixed